fla/ops/deltaformer/naive.py

```
@@ -36,16 +36,34 @@ def tril_softmax(scores: torch.Tensor, strict: bool = True) -> torch.Tensor:
     return probs
 
 
-def naive_deltaformer_attn(
+def naive_causal_attention_bhtd(
+    q: torch.Tensor,
+    k: torch.Tensor,
+    v: torch.Tensor,
+) -> torch.Tensor:
+    B, H, T, D = q.shape
```

Ruff (0.13.1)

Unpacked variable `B` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)